addon/adapters/build-url-mixin.js

```diff
@@ -49,7 +49,7 @@ export default Ember.Mixin.create({
     @param {Object} query object of query parameters to send for query requests.
     @return {String} url
   */
-  buildURL: function(modelName, id, snapshot, requestType, query) {
+  buildURL(modelName, id, snapshot, requestType, query) {
```

Comment:
I'm not a big fan of the concise method definitions because I find it makes it harder to search for the method definition in a file. 

That said, if others prefer this syntax I don't want to stand in the way of it being merged. cc @igorT @wecc @fivetanley